src/Bootstrapper/Controllers/Identity/TokensController.cs

```diff
@@ -33,7 +33,7 @@ public async Task<IActionResult> GetTokenAsync(TokenRequest request, [FromHeader
         [HttpPost("refresh")]
         [AllowAnonymous]
         [SwaggerHeader("tenant", "Input your tenant Id to access this API", "", true)]
-        public async Task<ActionResult> RefreshAsync(RefreshTokenRequest request, [FromHeader(Name = "tenant")][Required] string tenant = null)
```

Comment:
I guess this was mainly done earlier to facilitate adding tenant Key to request header via Swagger. Can you please check if this is not needed anymore, if someone wants to test the API via swagger ( token generation endpoint) ?

Reply:
that parameter is not being used and in the case of swagger it seems to be that it does not give problems. removing that parameter is necessary when looking for the tenant From Query String

![image](https://user-images.githubusercontent.com/22311691/143505288-341d82c9-4c41-4106-8e3b-9ef8ac82dfb6.png)